src/content-handlers/iiif/modules/uv-shared-module/HeaderPanel.ts

```diff
@@ -51,8 +51,12 @@ export class HeaderPanel<
     this.$rightOptions = $('<div class="rightOptions"></div>');
     this.$options.append(this.$rightOptions);
 
-    //this.$helpButton = $('<a href="#" class="action help">' + this.content.help + '</a>');
-    //this.$rightOptions.append(this.$helpButton);
+    this.$helpButton = $(`
+      <button class="btn imageBtn help" tabindex="0" title="${this.content.help}">
```

Comment:
Is there a strong reason to make this a button instead of a regular link? I imagine it may be useful for stylistic consistency, but asking in case there are accessibility disadvantages (e.g., this isn't really semantically button-like) that we might need to worry about.